Mathlib/Algebra/DirectSum/Basic.lean

```diff
@@ -131,6 +131,18 @@ theorem of_eq_of_ne (i j : ι) (x : β i) (h : i ≠ j) : (of _ i x) j = 0 :=
 lemma of_apply {i : ι} (j : ι) (x : β i) : of β i x j = if h : i = j then Eq.recOn h x else 0 :=
   DFinsupp.single_apply
 
+variable {β} in
```

Comment:
Probably we just want `variable {β}` starting from before `of_eq_same`. I think lots of these theorems have it too explicitly.